resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,6 +248,83 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class AnimationEffects {
+	#Need to verify the Animation Effects setting toggle on the Accessibility page changes when these are updated, or find the setting to update that also.
+    [DscProperty(Key)] [BinarySettingState] $SmoothScrollListBoxes = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $SlideOpenComboBoxes = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOrSlideMenusIntoView = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderMousePointer = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOrSlideToolTipsIntoView = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOutMenuItemsAfterClicking = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderWindows = [BinarySettingState]::KeepCurrentValue
+    [DscProperty(NotConfigurable)] [AnimationEffects] $currentState
+
+    [AnimationEffects] Get() {
+        $this.currentState = [AnimationEffects]::new()
+
+		$this.currentState = "Disabled"
+
+		foreach ($enum in [AnimationEffectsSettings].GetEnumNames()) {
+			$thisState = Get-AnimationState $enum
+			if ($thisState -eq [BinarySettingState]::Enabled) {
+				$this.currentState = "Enabled"
+			}
+		}
+
+		return $this.currentState
+	}
+
+    [bool] Test() {
+		$this.currentState = $this.Get()
+		if ($this.SmoothScrollListBoxes -ne [BinarySettingState]::KeepCurrentValue -and
+		$this.SmoothScrollListBoxes -ne $this.currentState.SmoothScrollListBoxes -and
+		 $this.SlideOpenComboBoxes -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.SlideOpenComboBoxes -ne $this.currentState.SlideOpenComboBoxes -and
+		 $this.FadeOrSlideMenusIntoView -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.FadeOrSlideMenusIntoView -ne $this.currentState.FadeOrSlideMenusIntoView -and
+		 $this.ShowShadowsUnderMousePointer -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.ShowShadowsUnderMousePointer -ne $this.currentState.ShowShadowsUnderMousePointer -and
+		 $this.ShowShadowsUnderMousePointer -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.ShowShadowsUnderMousePointer -ne $this.currentState.ShowShadowsUnderMousePointer -and
+		 $this.FadeOrSlideToolTipsIntoView -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.FadeOrSlideToolTipsIntoView -ne $this.currentState.FadeOrSlideToolTipsIntoView -and
+		 $this.FadeOutMenuItemsAfterClicking -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.FadeOutMenuItemsAfterClicking -ne $this.currentState.FadeOutMenuItemsAfterClicking -and
+		 $this.ShowShadowsUnderWindows -ne [BinarySettingState]::KeepCurrentValue -and
+		 $this.ShowShadowsUnderWindows -ne $this.currentState.ShowShadowsUnderWindows) {

```

Comment:
I think between properties you will need to use -or. 

Lets say $this.SmoothScrollListBoxes = [BinarySettingState]::KeepCurrentValue, then the rest of the comparisons won't be calculated since the condition statement will always be false. 

It's also okay to break it up by property and return false early if that is easier to read.